server/controllers/applicationsController.ts

```diff
@@ -174,10 +174,48 @@ const updateApplication = async (
   }
 };
 
+const getAggregatedUserStats = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const userId = req.query.user_id;
```

Comment:
For consistency, are we doing queries or parameters for user_id across codebase?